inc/html.class.php

```diff
@@ -6688,74 +6692,57 @@ public static function compileScss($args) {
          );
       }
 
-      if (!isset($args['file']) || $args['file'] == 'main_styles') {
-         $files[] = 'css/styles';
-         if (isset($_SESSION['glpihighcontrast_css'])
-            && $_SESSION['glpihighcontrast_css']) {
-            $ckey .= '_highcontrast';
-            $files[] = 'css/highcontrast';
-         }
+      $file = isset($args['file']) ? $args['file'] : 'css/styles';
 
-         // CSS theme
-         $theme = 'auror';
-         if (isset($_SESSION["glpipalette"])) {
-            $theme = $_SESSION['glpipalette'];
-         }
-         $ckey .= '_' . $theme;
-         $files[] = 'css/palettes/' . $theme;
-      } else {
-         $ckey .= '_' . md5($args['file']);
+      $ckey .= '_' . md5($file);
+      $ckey = 'css_' . md5($ckey);
```

Comment:
This double md5 seems to be overkill, maybe one only is sufficient now

Reply:
Fixed.